site/gatsby-site/src/components/discover/hitTypes/Details.js

```diff
@@ -38,14 +39,20 @@ export default function Details({ item, toggleFilterByIncidentId, viewType }) {
       <input type="hidden" data-cy="date-submitted" value={item.epoch_date_submitted} />
       <input type="hidden" data-cy="incident-date" value={item.epoch_incident_date} />
       <a href={detailsPath}>
-        <Image
-          className={`card-img-top rounded-t-lg h-[240px] object-cover w-full`}
-          publicID={item.cloudinary_id ? item.cloudinary_id : `legacy/${md5(item.image_url)}`}
-          alt={item.title}
-          transformation={fill().height(480)}
-          itemIdentifier={t('Report {{report_number}}', {
-            report_number: item.report_number,
-          }).replace(' ', '.')}
+        <VideoPlayer
+          className="card-img-top"
+          mediaURL={item.media_url}
+          fallback={
+            <Image
+              className={`card-img-top rounded-t-lg h-[240px] object-cover w-full`}
+              publicID={item.cloudinary_id ? item.cloudinary_id : `legacy/${md5(item.image_url)}`}
+              alt={item.title}
+              transformation={fill().height(480)}
+              itemIdentifier={t('Report {{report_number}}', {
+                report_number: item.report_number,
+              }).replace(' ', '.')}
```

Comment:
Is `itemIdentifier` field needed? It looks like `<Image>` component doesn't have a field with that name:
<img width="853" alt="image" src="https://github.com/responsible-ai-collaborative/aiid/assets/6564809/3d73c5fb-b4b6-4d6b-8ff1-4ee326fc0a06">

Reply:
From a search of the codebase, all other instances of `<Image>` also have `itemIdentifier`, so we kept it for consistency.